OpenSim/Simulation/Wrap/WrapObject.cpp

```diff
@@ -152,10 +154,10 @@ int WrapObject::wrapPathSegment(const SimTK::State& s,
     // Convert the path points from the frames of the bodies they are attached
     // to, to the frame of the wrap object's body
     pt1 = aPoint1.getBody()
-        .findStationLocationInAnotherFrame(s, aPoint1.getLocation(), getFrame());
+        .findStationLocationInAnotherFrame(s, aPoint1.getLocation(s), getFrame());
     
     pt2 = aPoint2.getBody()
-        .findStationLocationInAnotherFrame(s, aPoint2.getLocation(), getFrame());
+        .findStationLocationInAnotherFrame(s, aPoint2.getLocation(s), getFrame());
```

Comment:
I was not addressing a bug here, but since `WrapObject` always deal with `PathWrapPoint`s at least implicitly, this likely never encounters a `MovingPathPoint`.